plugin/main/src/kotlinx/benchmark/gradle/Utils.kt

```diff
@@ -256,9 +256,5 @@ internal val Gradle.isConfigurationCacheAvailable
     } ?: false
 
 internal fun Project.getSystemProperty(key: String): String? {
-    return if (gradle.isConfigurationCacheAvailable) {
```

Comment:
Is the `Gradle.isConfigurationCacheAvailable` property used anywhere else? If not, the property declaration above can be removed as well?